src/wp-content/themes/twentytwentytwo/inc/patterns/hidden-comments.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Comments block (for WP >= 6.1), or legacy Post Comments block.
```

Comment:
We'll need to update these since the PR didn't make it into WP 6.1.

```suggestion
 * Comments block (for WP >= 6.2), or legacy Post Comments block.
```